CI/codespell_ignore.txt

```diff
@@ -19,3 +19,4 @@ strack
 te
 vart
 writet
+interprete
```

Comment:
Hard to enforce, but I tried to keep the list alphabetical 🙈

Reply:
Oh I hadn't seen that; sorry! Let me update it.

Reply:
Now in alphabetical order. :)